CRM/ACL/BAO/Cache.php

```diff
@@ -149,14 +149,11 @@ public static function resetCache() {
     $config = CRM_Core_Config::singleton();
     $smartGroupCacheTimeout = CRM_Contact_BAO_GroupContactCache::smartGroupCacheTimeout();
 
-    //make sure to give original timezone settings again.
-    $now = CRM_Utils_Date::getUTCTime();
-
     $query = "
 DELETE
 FROM   civicrm_acl_cache
 WHERE  modified_date IS NULL
-   OR  (TIMESTAMPDIFF(MINUTE, modified_date, $now) >= $smartGroupCacheTimeout)
+   OR  (TIMESTAMPDIFF(MINUTE, modified_date, NOW()) >= $smartGroupCacheTimeout)
```

Comment:
Doesn't keeping this make it an unindexed query? Something we were trying to get rid of here: https://github.com/civicrm/civicrm-core/pull/7906/files#diff-e71b99945ad53aa0cfbd9efd2dbfc168L78

We can calculate that in PHP instead? Or is that out of scope for this PR?